### PR TITLE
Refactor job-handlers

### PR DESCRIPTION
**Pull request by tomaszdurka**
- introduce config entries for job-handler commands

Will be moved to different issue:
- create tmp directory for each job-handler processing
- create tmp files and run scripts in those tmp directories

### CODE REVIEW

File: lib/job/model/audioroom-recording.js [hunk 1]
```diff
@@ -44,14 +44,14 @@ AudioroomRecordingJob.prototype._run = function(tmpDir) {
...
+          unlink(webmFile, errorHandler);
           unlink(audioFilename, errorHandler);
         });
     });
 };
 
-AudioroomRecordingJob.prototype._audioConvert = function(audioFilename, mpegFilename) {
-  return this._runJobScript('audioroom-convert.sh', [audioFilename, mpegFilename]);
+AudioroomRecordingJob.prototype._audioConvert = function(audioFilename, webmFile) {
+  return this._runJobScript('mjr2webm', ['-a', audioFilename, webmFile]);
```
Comment (tomaszdurka) [inline lib/job/model/audioroom-recording.js]: @njam will this job-handler receive mjr or wav file?

**Comment by tomaszdurka** [comment]
@vogdb please review

@njam is this command configuration enough for your needs?
https://github.com/cargomedia/cm-janus/pull/117/files#diff-0e9b471e784ef9c3c8411a6a50a07d87

**Comment by njam** [comment]
yes, templating looks good!

**Comment by vogdb** [comment]
started the review

**Comment by vogdb** [comment]
Tests are failing. Use `mergeCommand` and `createThumbnailCommand` in `test/rtpbroadcast-recording` and `test/rtpbroadcast-thumbnail` correspondingly.

**Comment by vogdb** [comment]
lgtm

**Pull request merged**